### PR TITLE
Version 2

### PR DESCRIPTION
**Pull request by Incei**

### CODE REVIEW

**Comment by Incei** [comment]
@AndreDlm

File: Input/Qualification-Ontogeny-UGT1A1.json [hunk 1]
```diff
@@ -8,28 +8,18 @@
         },
         {
             "Id": "Raltegravir-Pediatrics",
-            "Path": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Raltegravir-Pediatrics/v1.0/Raltegravir-Pediatrics.json",
+            "Path": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Raltegravir-Pediatrics/main/Raltegravir-Pediatrics.json",
```
Comment (Yuri05) [inline Input/Qualification-Ontogeny-UGT1A1.json:11]: replace with the link to the latest **released** version
https://raw.githubusercontent.com/Open-Systems-Pharmacology/Raltegravir-Pediatrics/v1.0/Raltegravir-Pediatrics.json

File: Input/Qualification-Ontogeny-UGT1A1.json [hunk 2]
```diff
@@ -8,28 +8,18 @@
...
-                    "Project": "Raltegravir"
-                }
             ]
         }
     ],
     "ObservedDataSets": [
         {
             "Id": "PK-Parameter",
-            "Path": "https://raw.githubusercontent.com/Incei/Database-for-observed-data/master/Pediatrics.csv",
+            "Path": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Database-for-observed-data/master/Pediatrics.csv",
```
Comment (Yuri05) [inline Input/Qualification-Ontogeny-UGT1A1.json:19]: Replace with the link to the latest **released** version

File: Workflow.R [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+  }
+  
+  #' Activate/Deactivate tasks of qualification workflow prior running
+    #workflow$inactivateTasks("simulate")
+    #workflow$inactivateTasks("calculatePKParameters")
+    #workflow$inactivateTasks("plotTimeProfiles")
+    workflow$inactivateTasks("plotComparisonTimeProfile")
+    workflow$inactivateTasks("plotGOFMerged")
+    #workflow$inactivateTasks("plotPKRatio")
+    workflow$inactivateTasks("plotDDIRatio")
```
Comment (Yuri05) [inline Workflow.R:150]: Don't deactivate tasks.

File: Workflow.R [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+  
+  #' Run the `QualificatitonWorklfow`
+   #workflow$plotPKRatio$settings$units$<PK parameter name> <- <PK parameter unit>
+   #workflow$plotPKRatio$settings$units$C_max <- "ng/mL"
```
Comment (Yuri05) [inline Workflow.R:154]: remove unneeded code

**Review by Yuri05 — CHANGES_REQUESTED** [review]
s. comments

File: Input/Qualification-Ontogeny-UGT1A1.json (hunk 2 above)
Comment (Incei) [reply Input/Qualification-Ontogeny-UGT1A1.json:19]: We first need a new release, as the pediatrics.csv needs to be released. For now i include the latest release.

File: Input/Content/Translation.md [hunk 5]
```diff
@@ -6,7 +6,7 @@ The models were evaluated by ratio plots of area under the plasma concentration-
 
 
 
-Eq. 1: GMFE=10^((Σ|log10(pred PK parameter∕obs PK parameter)|)∕n)
+Eq. 1: $$ {\Huge  GMFE = 10^{\frac{\sum(|log(\frac{Pred}{Obs})|)}{n}} }  $$
```
Comment (Yuri05) [inline Input/Content/Translation.md:9]: Replace `$$ `with `$`.
When having additional text in the same line (here: `Eq. 1:`), $ must be used
Already created the same issue for 1A9: https://github.com/Open-Systems-Pharmacology/Pediatric_Qualification_Package_UGT1A9_Ontogeny/issues/8

**Pull request merged**